languages/elixir/exercises/concept/anonymous-functions/.docs/introduction.md

```diff
@@ -0,0 +1,65 @@
+Functions are treated as first class citizens in Elixir. This means functions:
+
+- Can be assigned to variables.
+- Can be passed around like data as arguments and return values.
+- Can be created dynamically.
+
+```elxir
+varaible = fn param ->
+  param + 1
+end
+```
+
+Anonymous function start with the reserved word `fn`, the parameters are separated from the body of the function with the `->` token, and they are finished with an `end`. As with named functions, the last expression in the function is _implicitly returned_ to the calling function.
+
+To invoke a function reference, you must use a `.` between the reference variable and the list of parameters:
+
+```elixir
+varaible = fn param ->
+  param + 1
+end
+
+variable.(1)
+# => 2
+```
+
+You can even use short hand capture notation to make this more concise:
+
+```elixir
+variable = &(&1 + 1)
+
+variable.(1)
+# => 2
+```
+
+Which can be useful when are using other functions:
```

Comment:
Is this something that can perhaps be moved to the `after.md` file (or is this required to solve the exercise)?